integration-test/https-loop-protection.spec.js

```diff
@@ -0,0 +1,27 @@
+import { test, expect } from './helpers/playwrightHarness'
+import backgroundWait from './helpers/backgroundWait'
+import { routeFromLocalhost } from './helpers/testPages'
+
+const loopProtectionPage = 'https://good.third-party.site/privacy-protections/https-loop-protection/'
+
+test.describe('Loop protection', () => {
+    test('Loop protection page should prevent loading https:// infinately', async ({ context, backgroundPage, page }) => {
+        await backgroundWait.forExtensionLoaded(context)
+        await backgroundWait.forAllConfiguration(backgroundPage)
+        await routeFromLocalhost(page)
+
+        await page.goto(loopProtectionPage, { waitUntil: 'networkidle' })
+        await page.click('#start')
+        await page.waitForFunction(
+            () => results.date !== null && results.results[0].value !== null,
+            { polling: 100, timeout: 60000 }
```

Comment:
This timeout seems far too long, but I see you took this from the previous file. I'll leave it up to you.

Reply:
Good spot. I reduced it a bit.